### DISCUSSION TITLE
Upgrade to scala 3?

### DISCUSSION THREAD

**Discussion by costa100** [gvolpe/pfps-shopping-cart#691] [Q&A] [answered]
Hi,

I was wondering if you have any plans to upgrade the projects from the Practical FP in Scala book to Scala 3. Maybe another edition of the book?

Thanks

**Comment by gvolpe** [comment, marked as answer]
No plans to do so. My newest book covers Scala 3 and what's needed to migrate any application that uses the same libraries as `pfps-shopping-cart` to this new version of the language: https://leanpub.com/feda
